### DISCUSSION TITLE
Is it possible for "drag start" to be triggered twice?

### DISCUSSION THREAD

**Discussion by staticshock** [adobe/react-spectrum#6845] [Q&A]
I'm seeing this DnD error in Sentry (Chrome 127 / Windows 10), and I'm at a loss as to how to reproduce it:

> NotSupportedError
> Failed to execute 'add' on 'DataTransferItemList': An item already exists for type 'text/plain'.

![Screenshot 2024-08-07 at 2 19 58 PM](https://github.com/user-attachments/assets/b443fa4d-e9a2-447a-a3f7-784506f09fb9)

The error seems to have happened on drag start.

My draggable metadata is defined like this:

```jsx
const { dragAndDropHooks } = useDragAndDrop({
  getItems: (keys) =>
    members
      .filter((member) => keys.has(member.id))
      .map((member) => ({
        'text/plain': member.id,
      })),
  ...
})

return <ListBox dragAndDropHooks={dragAndDropHooks} …>…</ListBox>
```

Anyone have suggestions for how to troubleshoot? I'm not even sure how to contort my app code, even in theory, to force it to fail in this way.

**Comment by snowystinger** [comment]
Not that i'm aware of, I assume you've tried reproducing this in a codesandbox or stackblitz? can you share a link here? it's ok if it doesn't reproduce, it'll let us see more of what you're doing

**Comment by VapidLinus** [comment]
@staticshock Did you find a fix for this? I am also seeing this problem

**Comment by staticshock** [comment]
No, but I just took a look at some replays for this issue and noticed that just before the issue, some text is always selected on the screen, possibly indicating a failed drag preceding the drag that throws the error.

**Reply by VapidLinus** [reply]
Huh you're right. You can even get this on the official docs website! If you make a selection that goes "beyond" the drag item, once you start dragging it, you will see the "" error in the console!

![mousemaster_ho8gNMFhrs](https://github.com/user-attachments/assets/a1bc16d1-ba18-4e8c-ae86-cc87d642eec2)

https://react-spectrum.adobe.com/react-aria/useDrag.html

**Reply by VapidLinus** [reply]
I created an issue for it https://github.com/adobe/react-spectrum/issues/8672
